src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -1959,7 +1953,7 @@ private bool ShowWarningDialogOnSaveWithUnresolvedIssues()
             if (args.ClickedButtonId == (int)DynamoModel.ButtonId.Cancel ||
                 args.ClickedButtonId == 0)
             {
-                OnRequestOpenLinterView();
+                OnViewExtensionOpenRequest("Graph Status");
```

Comment:
can you use name of?

Reply:
This one we can't do without adding a dependency to the specific ViewExtension to DynamoCoreWPF project

Reply:
ah, I see, let's not do that.